src/content/blog/2021/12/17/react-conf-2021-recap.md

```diff
@@ -95,7 +95,7 @@ Looking further into the future, [Xuan Huang (黄玄)](https://twitter.com/Huxpr
 
 ## React docs keynote {/*react-docs-keynote*/}
 
-[Rachel Nabors](https://twitter.com/rachelnabors) kicked off a section of talks about learning and designing with React with a keynote about our investment in React's [new docs](https://beta.reactjs.org/):
+[Rachel Nabors](https://twitter.com/rachelnabors) kicked off a section of talks about learning and designing with React with a keynote about our investment in React's [new docs](https://react.dev/):
```

Comment:
I'd change that to "React's new docs ([now shipped as react.dev](https://react.dev/blog/2023/03/16/introducing-react-dev))"

Reply:
done.